mods/d2k/languages/rules/en.ftl

```diff
@@ -280,287 +280,287 @@ actor-deathhand =
+    Even with heavy armor, the Starport is vulnerable to most weapons
 
 actor-wall =
-   .description = Stop units and blocks enemy fire.
+   .description = Stops units and blocks enemy fire
    .name = Concrete Wall
    .generic-name = Structure
-   .encyclopedia = Base defense. Concrete walls are the most effective barriers on Dune. Concrete walls will block tank bullets and impede unit movement.
+   .encyclopedia = Base defense: Concrete walls are the most effective barriers on Dune, effectively blocking tank bullets and hindering unit movement
 
-    Walls can only be damaged by explosive weapons, missiles and shells. Like concrete slabs, walls cannot be repaired if damaged.
+    Walls can only be damaged by explosive weapons, missiles and shells. Similar to concrete slabs, they cannot be repaired once damaged.
 
 actor-medium-gun-turret =
-   .description = Defensive structure.
-      Strong vs Tanks
-      Weak vs Infantry, Aircraft
+   .description = Defensive structure
+      Strong vs. Tanks
+      Weak vs. Infantry and Aircraft
    .name = Gun Turret
-   .encyclopedia = The Gun Turret has a medium range gun which is effective against vehicles, especially heavily armored vehicles. The Gun Turret will fire on any enemy unit within range.
+   .encyclopedia = The Gun Turret has a medium-range weapon that is highly effective against vehicles, including heavily armored ones. It will automatically fire upon any enemy unit within its range.
 
-    The Gun Turret is resistant to bullet and explosive weapons, but vulnerable to missiles and high-caliber guns.
+    The Gun Turret is resistant to firearms and explosive weapons, but vulnerable to missiles and high-caliber guns
 
 actor-large-gun-turret =
-   .description = Defensive structure.
-      Strong vs Infantry, Aircraft
-      Weak vs Tanks
+   .description = Defensive structure
+      Strong vs. Infantry and Aircraft
+      Weak vs. Tanks
 
-      Requires power to operate.
+      Requires power to operate
    .name = Rocket Turret
-   .encyclopedia = The substantially improved Rocket Turret has a longer range and a higher rate of fire than the Gun Turret. The Rocket Turret's advanced targeting equipment requires power to operate.
+   .encyclopedia = The enhanced Rocket Turret boasts longer range and a higher rate of fire compared to the Gun Turret. Its advanced targeting system requires power to operate.
 
-    The Rocket Turret is resistant to bullet and explosive weapons, but vulnerable to missiles and high-caliber guns.
+    The Rocket Turret is resistant to firearms and explosive weapons, but vulnerable to missiles and high-caliber guns
 
 actor-repair-pad =
-   .description = Repairs vehicles.
-     Allows construction of MCVs
+   .description = Repairs vehicles
+     Allows MCVs to be built
    .name = Repair Pad
-   .encyclopedia = With a Repair Pad, vehicles can be repaired for a varying price.
+   .encyclopedia = Repair Pads repair units for up to 20% of the units cost
```

Comment:
This wording reads to me as if the repair cap is 20%. Is it instead trying to state that repairs are 5x cheaper than a new unit?